demo/app/components/chip/chip.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, ViewChild, AfterViewInit, QueryList, ElementRef, TemplateRef } from '@angular/core';
+import {
+  TsChipCollectionComponent,
+  TsChipComponent,
+ } from '@terminus/ui/chip';
+
+@Component({
+  selector: 'demo-chip',
+  templateUrl: './chip.component.html',
+})
+export class ChipComponent implements OnInit {
+
+  options = ['banana', 'apple', 'orange', 'pear'];
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  removingOption(value: string) {
+    const ctrlValue = this.options;
+    const index = ctrlValue.indexOf(value);
+    const selections = ctrlValue.slice();
+    // If not found
+    if (index < 0) {
+      return;
+    }
+    selections.splice(index, 1);
+    this.options = selections;
+  }
+
+  change(value: string | string[]) {
+    console.log('collection change triggered value: ', value);
```

Comment:
Let's prefix this message like we do with our other demo logs: `DEMO: my message`